Description: Add soname to library.
Author: Monty Taylor <mordred@inaugust.com>
Last-Update: 2011-01-08
--- a/Makefile
+++ b/Makefile
@@ -111,7 +111,7 @@
 	$(CXX) -dynamiclib -Wl,-install_name,$(LIBSONAME) -o $(LIBFILENAME) $(LIBOBJECTS) -lc
 else
 	#GNU/Linux case
-	$(CXX) -shared -Wl,-soname,$(LIBSONAME) -o $(LIBFILENAME) $(LIBOBJECTS) -lc
+	$(CXX) -shared -Wl,-soname,$(LIBSONAME).1 -o $(LIBFILENAME) $(LIBOBJECTS) -lc
 endif
 
 lib: $(LIBFILENAME)
